python/py-astroML/Portfile

```diff
@@ -8,6 +8,7 @@ set realname        astroML
 
 name                py-${realname}
 version             0.3
+revision            1
```

Comment:
No need for revbump.

Reply:
I think it's required here as the installed files change. The default variant is ```+addons```, which in the original Portfile incorrectly always installed ```py-astroML_addons``` instead of the correct subport.

Reply:
ah!